test/unit/mdc-snackbar/foundation.test.js

```diff
@@ -74,6 +77,30 @@ test('#destroy calls adapter.deregisterActionClickHandler() with a registerActio
   td.verify(mockAdapter.deregisterActionClickHandler(changeHandler));
 });
 
+test('#destroy calls adapter.deregisterVisibilityChangeHandler() with a function', () => {
+  const {foundation, mockAdapter} = setupTest();
+  const {isA} = td.matchers;
+
+  foundation.destroy();
+  td.verify(mockAdapter.deregisterVisibilityChangeHandler(isA(Function)));
```

Comment:
However, this is `destroy()` but `init()` 😃 . I can see totally why we won't test it in `init()`